internal/services/formatters/csharp/dotnet_cli/formatter.go

```diff
@@ -154,22 +154,31 @@ func (f *Formatter) parseFieldByIndex(index int, fieldValue string, dependency *
 func (f *Formatter) newVulnerability(dependency *dotnetDependency,
 	projectSubPath string) (*vulnerability.Vulnerability, error,
 ) {
-	code, filePath, line, err := f.getCodeFilePathAndLine(dependency, projectSubPath)
+	code, absFilePath, line, err := f.getCodeFilePathAndLine(dependency, projectSubPath)
 	if err != nil {
 		return nil, err
 	}
+
+	relFilePath, err := filepath.Rel(f.GetConfigProjectPath(), absFilePath)
+	if err != nil {
+		return nil, err
+	}
+
 	vuln := &vulnerability.Vulnerability{
 		SecurityTool: tools.DotnetCli,
 		Language:     languages.CSharp,
 		Confidence:   confidence.High,
-		RuleID:       vulnHash.HashRuleID(dependency.getDescription()),
+		RuleID:       vulnhash.HashRuleID(dependency.getDescription()),
 		Details:      dependency.getDescription(),
 		Code:         code,
-		File:         f.removeHorusecFolder(filePath),
+		File:         relFilePath,
 		Line:         line,
 		Severity:     dependency.getSeverity(),
 	}
-	return f.SetCommitAuthor(vulnHash.Bind(vuln)), nil
+
+	vuln.DeprecatedHashes = f.getDeprecatedHashes(absFilePath, *vuln)
```

Comment:
Why do we need this if vulnhash.Bind already set the deprecated hashes?

Reply:
The hash was being generated with an absolute path, with the change to the relative one, it was necessary to add the hashes generated with the absolute path as well.


The getDeprecatedHashes function has a comment that explains this.

Reply:
Make totally sense, I was not understanding the needed to generate the hash twice.